### PR TITLE
Removed consul_telemetry_dogstatsd_addr: true from variables  and  removed static values from consul.json.j2 template

### PR DESCRIPTION
**Pull request by swetha9devops**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,8 @@ consul_statsite_address: "127.0.0.1:8125"
 consul_statsite_prefix: "consul"
 # if you don't want to prepend runtime telemetry with the machine's hostname (consul 0.6.4 or later)
 consul_telemetry_disable_hostname: true
+# if you want Consul to send metrics to a datadog
+consul_telemetry_dogstatsd_address: "127.0.0.1:8125"
```
Comment (trevorr) [inline README.md:219]: Did you mean to comment this out? It shouldn't be enabled by default.

File: templates/consul.json.j2 [hunk 2]
```diff
@@ -144,11 +144,14 @@
 {% if consul_statsite_prefix is defined %}
   "statsite_prefix": "{{ consul_statsite_prefix }}",
 {% endif %}
+"telemetry": {
```
Comment (trevorr) [inline templates/consul.json.j2]: I think this should retain the original indentation (2 spaces).

**Review by trevorr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (swetha9devops) [reply README.md:219]: I think it is not enabled by default. Whenever any user want to use this consul_telemetry_dogstatsd_address, he can define it in the variables section.

Comment (trevorr) [reply README.md:219]: Sorry, I was thinking this was the defaults/main.yml. This is fine.

**Review by trevorr — APPROVED** [review]

**Pull request merged**